src/templates/base.html

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
 
+{% load static %}
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="">
-    <link rel="icon" type="image/png" sizes="96x96" href="img/favicon.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="{% static 'problems/img/favicon.png' %}">
 
-    <title>404 - Project Lovelace</title>
+    <title>Finding earthquake epicenters | Project Lovelace</title>
 
     <!-- Bootstrap core CSS with Jumbotron Theme -->
     <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```

Comment:
Ah never mind, didn't scroll to the right lol. Then I'd go for non-minified version here too.